modules/ztp-setting-managed-bare-metal-host-kernel-parameters-manual-workflow.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-parameters_{context}"]
+= Setting managed bare-metal host kernel parameters
+
+Specify kernel parameters for managed bare-metal hosts to customize the host's boot process. This is useful for clusters with specific environmental requirements. For example, specify the `rd.net.timeout.carrier` kernel parameter to facilitate static networking or to receive a DHCP address before downloading the root file system. 
+
+[NOTE]
+====
+In {product-title} {product-version}, you can only add kernel parameters. You can not replace or delete kernel parameters.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+* You have manually generated the installation and configuration custom resources (CRs).
+
+.Procedure
+
+. Edit the `spec.kernelArguments` specifications in the `InfraEnv` CR to add kernel values:
+
+[source,yaml,options="nowrap",role="white-space-pre"]
+---
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  name: <cluster_name>
+  namespace: <cluster_name>
+spec:
+  kernelArguments:
+    - operation: append
+      value: audit=0
+    - operation: append
+      value: trace=1
+  clusterRef:
+    name: <cluster_name>
+    namespace: <cluster_name>
+  pullSecretRef:
+    name: pull-secret
+---
+
+[NOTE]
+====
+The `SiteConfig` CR generates the `InfraEnv` resource as part of the day-0 installation CRs.
+====
+
+.Verification
+Any verification steps to verify that the kernel parameters are applied?
+
```

Comment:
@ori-amizur - are there any verification steps that can be done to check that the new kernel parameters are applied?

I think this procedure is ok for the "manual" workflow. I will work on the procedure for the "gitops" workflow now as Ian Miller gave some good info on Slack about that approach.